elodie.py

```diff
@@ -181,8 +183,22 @@ def update_location(media, file_path, location_name):
             print(('{"source":"%s",' % file_path,
                 '"error_msg":"Failed to update location"}'))
             sys.exit(1)
-    return True
 
+        config = load_config()
+        if ('Location' in config) and (config['Location'].getboolean('write_location')):
```

Comment:
Not sure I follow you... I was thinking the idea was to not always add the tags, if that config option isn't set. Otherwise I don't need the config option.